cypress/e2e/RandomView_spec.cy.js

```diff
@@ -39,11 +44,10 @@ describe("small Talk landing page (Random View)", () => {
     cy.get(".booster-card").contains("Mix together the amaretto and orange juice. Pour into glass and then add the grenadine untill you see the sunrise")
   });
 
-  it("should give me a new drink when I click the new drink button", () => {
-    cy.get("button").last().click()
-    cy.get(".booster-card").contains("Amaretto Sunrise")
-    cy.get(".booster-card").contains("1 cl Amaretto")
-    cy.get(".booster-card").contains("Mix together the amaretto and orange juice. Pour into glass and then add the grenadine untill you see the sunrise")
+  it("should give me a new joke when I click the new joke button", () => {
+    cy.get("button").eq(1).click()
+    cy.get(".booster-card").contains("joke")
+    cy.get(".booster-card").contains("Why did the octopus beat the shark in a fight? Because it was well armed.")
```

Comment:
Thank you so much for testing all the specific text to make sure it's showing up correctly Masaki!